src/store/phase.ts

```diff
@@ -1,34 +1,56 @@
 import { makeAutoObservable } from 'mobx';
 import { Ticket } from './interface';
 
-export interface IPhasePlannerStore {
+interface PhaseTickets {
+  [phase_uuid: string]: Ticket[];
+}
+
+export interface ITicketStore {
```

Comment:
Can we explain ITicketStore instead of TicketStore?

Ideally I'd prefer not to change names from design to avoid issues?

Reply:
@humansinstitute Fixed